bundles/org.smarthomej.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -521,6 +521,11 @@
 		<config-description-ref uri="thing-type:deconz:sensor"/>
 	</thing-type>
 
+	<channel-type id="locked">
+		<item-type>Switch</item-type>
+		<label>Locked</label>
+		<description>Status of this thermostat's childlock.</description>
```

Comment:
Maybe add category "Lock"? And define it as an advanced channel type as it is not available an every device?

```suggestion
		<description>Status of this thermostat's childlock.</description>
		<category>Lock</category>
```

Reply:
I also thought about that. But after thinking about it, I decided against it. Isn't the category for a "physical" lock (like a doorlock)? This is more a keylock. I didn't find a clear definition when to use the category.


Regarding advanced: since the channel is only added when it is actually available and there are still less than ten channels on the device, this is not needed on my opinion.

Reply:
The category has no real meaning on the technical side. It is just there to define a default icon shown in the UIs.